tests/robot-tests/tests/libs/common.robot

```diff
@@ -571,6 +571,14 @@ user waits until h3 is not visible
     [Arguments]    ${text}    ${wait}=${timeout}
     user waits until element is not visible    xpath://h3[text()="${text}"]    ${wait}
 
+user waits until h4 is visible
+    [Arguments]    ${text}    ${wait}=${timeout}
+    user waits until element is visible    xpath://h3[text()="${text}"]    ${wait}
+
+user waits until h4 is not visible
+    [Arguments]    ${text}    ${wait}=${timeout}
+    user waits until element is not visible    xpath://h3[text()="${text}"]    ${wait}
```

Comment:
I think there's a mistake in both of these here with h4/h3 because they are currently identical to the h3 variants.

Looks like they may also be unused by your changes.

Reply:
Indeed they are wrong AND unused!  Removed, but we can always add them in again if we ever need to.